File: www/Bon_plans_web_app/Entity/Utilisateur.php
<?php
/**
 * Created by PhpStorm.
 * User: SadfiAmine
 * Date: 15/04/2018
 * Time: 13:13
 */

class Utilisateur
{
    private $idUtilisateur;
    private $idGroupe;
    private $mdp;
    private $email;
    private $nom;
    private $prenom;
    private $urlphoto;
    private $ville;
    private $adresse;
    private $dateCreation;
    private $dateAcces;
    private $tempsAcces;

    public function __construct()
    {
    }
    public function __construct1($idUtilisateur,$idGroupe,$mdp,$email,$nom,$prenom,$urlphoto,$ville,$adresse,$dateCreation,$dateAcces,$tempsAcces)
    {   $this->$idUtilisateur=$idUtilisateur;
        $this->$idGroupe=$idGroupe;
        $this->$mdp=$mdp;
        $this->$email=$email;
        $this->$nom=$nom;
        $this->$prenom=$prenom;
        $this->$urlphoto=$urlphoto;
        $this->$ville=$ville;
        $this->$adresse=$adresse;
        $this->$dateCreation=$dateCreation;
        $this->$dateAcces=$dateAcces;
        $this->$tempsAcces=$tempsAcces;

    }
    public function __construct2($idGroupe,$mdp,$email,$nom,$prenom,$urlphoto,$ville,$adresse,$dateCreation,$dateAcces,$tempsAcces)
    {
        $this->$idGroupe=$idGroupe;
        $this->$mdp=$mdp;
        $this->$email=$email;
        $this->$nom=$nom;
        $this->$prenom=$prenom;
        $this->$urlphoto=$urlphoto;
        $this->$ville=$ville;
        $this->$adresse=$adresse;
        $this->$dateCreation=$dateCreation;
        $this->$dateAcces=$dateAcces;
        $this->$tempsAcces=$tempsAcces;
    }

    /**
     * @return mixed
     */
    public function getUrlphoto()
    {
        return $this->urlphoto;
    }

    /**
     * @param mixed $urlphoto
     */
    public function setUrlphoto($urlphoto)
    {
        $this->urlphoto = $urlphoto;
    }

    /**
     * @return mixed
     */
    public function getVille()
    {
        return $this->ville;
    }

    /**
     * @param mixed $ville
     */
    public function setVille($ville)
    {
        $this->ville = $ville;
    }

    /**
     * @return mixed
     */
    public function getPrenom()
    {
        return $this->prenom;
    }

    /**
     * @param mixed $prenom
     */
    public function setPrenom($prenom)
    {
        $this->prenom = $prenom;
    }

    /**
     * @return mixed
     */
    public function getNom()
    {
        return $this->nom;
    }

    /**
     * @param mixed $nom
     */
    public function setNom($nom)
    {
        $this->nom = $nom;
    }

    /**
     * @return mixed
     */
    public function getAdresse()
    {
        return $this->adresse;
    }

    /**
     * @param mixed $adresse
     */
    public function setAdresse($adresse)
    {
        $this->adresse = $adresse;
    }

    /**
     * @return mixed
     */
    public function getDateAcces()
    {
        return $this->dateAcces;
    }

    /**
     * @param mixed $dateAcces
     */
    public function setDateAcces($dateAcces)
    {
        $this->dateAcces = $dateAcces;
    }

    /**
     * @return mixed
     */
    public function getDateCreation()
    {
        return $this->dateCreation;
    }

    /**
     * @param mixed $dateCreation
     */
    public function setDateCreation($dateCreation)
    {
        $this->dateCreation = $dateCreation;
    }

    /**
     * @return mixed
     */
    public function getEmail()
    {
        return $this->email;
    }

    /**
     * @param mixed $email
     */
    public function setEmail($email)
    {
        $this->email = $email;
    }

    /**
     * @return mixed
     */
    public function getIdGroupe()
    {
        return $this->idGroupe;
    }

    /**
     * @param mixed $idGroupe
     */
    public function setIdGroupe($idGroupe)
    {
        $this->idGroupe = $idGroupe;
    }

    /**
     * @return mixed
     */
    public function getIdUtilisateur()
    {
        return $this->idUtilisateur;
    }

    /**
     * @param mixed $idUtilisateur
     */
    public function setIdUtilisateur($idUtilisateur)
    {
        $this->idUtilisateur = $idUtilisateur;
    }

    /**
     * @return mixed
     */
    public function getMdp()
    {
        return $this->mdp;
    }

    /**
     * @param mixed $mdp
     */
    public function setMdp($mdp)
    {
        $this->mdp = $mdp;
    }

    /**
     * @return mixed
     */
    public function getTempsAcces()
    {
        return $this->tempsAcces;
    }

    /**
     * @param mixed $tempsAcces
     */
    public function setTempsAcces($tempsAcces)
    {
        $this->tempsAcces = $tempsAcces;
    }
}

File: Bon_plans_desktop_app/src/tn/esprit/bonplans/entity/Utilisateur.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package tn.esprit.bonplans.entity;

import java.util.Date;
import java.sql.Timestamp;
import java.util.Objects;

/**
 *
 * @author KC
 */
public class Utilisateur {
    
    private int idUtilisateur;
    private int idGroupe;
    private String mdp;
    private String email;
    private String nom;
    private String prenom;
    private String urlphoto;
    private String ville;
    private String adresse;
    private Date dateCreation;
    private Date dateAcces;
    private Timestamp tempsAcces;
    private boolean isActif;
    private int codeActivation;

    public Utilisateur() {
    }
    
    public Utilisateur(int idUtilisateur, int idGroupe, String mdp, String email, String nom, String prenom, String urlphoto, String ville, String adresse, Date dateCreation, Date dateAcces, Timestamp tempsAcces, boolean isActif, int codeActivation) {
        this.idUtilisateur = idUtilisateur;
        this.idGroupe = idGroupe;
        this.mdp = mdp;
        this.email = email;
        this.nom = nom;
        this.prenom = prenom;
        this.urlphoto = urlphoto;
        this.ville = ville;
        this.adresse = adresse;
        this.dateCreation = dateCreation;
        this.dateAcces = dateAcces;
        this.tempsAcces = tempsAcces;
        this.isActif = isActif;
        this.codeActivation = codeActivation;
    }
    
    public Utilisateur (int idGroupe, String mdp, String email, String nom, String prenom, Date dateCreation, int codeActivation){
        this.idGroupe = idGroupe;
        this.mdp = mdp;
        this.email = email;
        this.nom = nom;
        this.prenom = prenom;
        this.dateCreation = dateCreation;
        this.isActif = false;
        this.codeActivation = codeActivation;
    }

    public int getCodeActivation() {
        return codeActivation;
    }

    public void setCodeActivation(int codeActivation) {
        this.codeActivation = codeActivation;
    }

    public boolean getIsActif() {
        return isActif;
    }

    public void setIsActif(boolean isActif) {
        this.isActif = isActif;
    }
    
    public int getIdUtilisateur() {
        return idUtilisateur;
    }

    public void setIdUtilisateur(int idUtilisateur) {
        this.idUtilisateur = idUtilisateur;
    }

    public int getIdGroupe() {
        return idGroupe;
    }

    public void setIdGroupe(int idGroupe) {
        this.idGroupe = idGroupe;
    }

    public String getMdp() {
        return mdp;
    }

    public void setMdp(String mdp) {
        this.mdp = mdp;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getPrenom() {
        return prenom;
    }

    public void setPrenom(String prenom) {
        this.prenom = prenom;
    }

    public String getUrlphoto() {
        return urlphoto;
    }

    public void setUrlphoto(String urlphoto) {
        this.urlphoto = urlphoto;
    }

    public String getVille() {
        return ville;
    }

    public void setVille(String ville) {
        this.ville = ville;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public Date getDateCreation() {
        return dateCreation;
    }

    public void setDateCreation(Date dateCreation) {
        this.dateCreation = dateCreation;
    }

    public Date getDateAcces() {
        return dateAcces;
    }

    public void setDateAcces(Date dateAcces) {
        this.dateAcces = dateAcces;
    }

    public Timestamp getTempsAcces() {
        return tempsAcces;
    }

    public void setTempsAcces(Timestamp tempsAcces) {
        this.tempsAcces = tempsAcces;
    }

    @Override
    public String toString() {
        return "Utilisateur{" + "idUtilisateur=" + idUtilisateur + ", idGroupe=" + idGroupe + ", mdp=" + mdp + ", email=" + email + ", nom=" + nom + ", prenom=" + prenom + ", urlphoto=" + urlphoto + ", ville=" + ville + ", adresse=" + adresse + ", dateCreation=" + dateCreation + ", dateAcces=" + dateAcces + ", tempsAcces=" + tempsAcces + ", isActif=" + isActif + ", codeActivation=" + codeActivation + '}';
    }

    @Override
    public int hashCode() {
        int hash = 3;
        hash = 17 * hash + this.idUtilisateur;
        hash = 17 * hash + Objects.hashCode(this.email);
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Utilisateur other = (Utilisateur) obj;
        if (this.idUtilisateur != other.idUtilisateur) {
            return false;
        }
        if (!Objects.equals(this.email, other.email)) {
            return false;
        }
        return true;
    }    

}
